### PR TITLE
[BUG] fix nightly is timing out

### LINKED ISSUE #811: [BUG] 0.3 nightly is timing out
**Describe the bug**
the 0.3 nightly build is timing out. Its also not reporting to the cicd channel on slack.

If we need to increase timeout lets do it and fix reporting. The reason I noticed was the integration test gets killed. Can we decouple those timeouts as well?

### PR DESCRIPTION
**Pull request by pxLi**
Signed-off-by: Peixin Li <pxli@nyu.edu>

fix #811 
also reporting issue should have been fixed within #805

### CODE REVIEW

**Comment by pxLi** [comment]
build

File: jenkins/Jenkinsfile.nightly [hunk 1]
```diff
@@ -48,7 +48,7 @@ pipeline {
     }
 
     triggers {
-        cron('H 5 * * *')
+        cron('H 10 * * *') // 10am PDT
```
Comment (jlowe) [inline jenkins/Jenkinsfile.nightly:51]: Curious why this was changed?

**Review by jlowe — APPROVED** [review]
Looks good to me, just had a question on why we moved the start time five hours rather than only increasing the timeout.

**Comment by tgravescs** [comment]
looks fine to me, same question as Jason

**Comment by pxLi** [comment]
> Looks good to me, just had a question on why we moved the start time five hours rather than only increasing the timeout.

Previously spark-nightly pipelines share nearly the same time slots (started at 6 PDT) as the rapids nightly, as NGCC only have 4 GPU nodes which could fail to schedule rapids-nightly-triggered integration tests, so that's why we were seeing timeout for rapids nightly and integrations tests sometimes

**Pull request merged**